docs/reference/migration/migrate_7_0/logging.asciidoc

```diff
@@ -3,8 +3,33 @@
 === Logging changes
 
 [float]
-==== Elasticsearch server's logs have been renamed from `${sys:es.logs.cluster_name}.log` to
-`${sys:es.logs.cluster_name}_server.log`.
+==== New JSON format log files in `log` directory
 
-Log files ending with `*.log` are now considered deprecated and newly added JSON logs ending
-`*.json` should be used.
+Elasticsearch now will produce additional log files in JSON format. They will be stored in `*.json` suffix files.
+Following files should be expected now in log directory:
+* ${cluster_name}_server.json
+* ${cluster_name}_deprecation.json
+* ${cluster_name}_index_search_slowlog.json
+* ${cluster_name}_index_indexing_slowlog.json
+* ${cluster_name}.log
+* ${cluster_name}_deprecation.log
+* ${cluster_name}_index_search_slowlog.log
+* ${cluster_name}_index_indexing_slowlog.log
+* ${cluster_name}_audit.json
+* gc.log
+* http.ports
+* transport.ports
+
+Note: You can override additional files by removing the unwanted appenders in `log4j2.properties`.
```

Comment:
Maybe: "You can configure which of these files are written by editing `log4j2.properties`"?